### PR TITLE
Make the DLR a submodule of IronPython

### PR DESCRIPTION
**Pull request by slide**

### CODE REVIEW

**Comment by slide** [comment]
This depends on https://github.com/IronLanguages/dlr/pull/9

File: Src/IronPythonTest/Cases/CaseExecuter.cs [hunk 1]
```diff
@@ -26,9 +26,30 @@ class CaseExecuter {
                 Executable,
                 "");
 
+            AddSearchPaths(engine);
             return engine;
         }
 
+        private static void AddSearchPaths(ScriptEngine engine) {
+            var paths = new List<string>(engine.GetSearchPaths());
+            if(!paths.Any(x => x.ToLower().Contains("StdLib"))) {
```
Comment (slozier) [inline Src/IronPythonTest/Cases/CaseExecuter.cs]: Isn't `x.ToLower().Contains("StdLib")` always `false`?

File: .gitattributes [hunk 2]
```diff
@@ -23,6 +23,10 @@
 *.vdproj text eol=crlf
 
 
+# Declare fiels that will always have LF line endings on checkout
```
Comment (slozier) [inline .gitattributes]: Typo: fiels

File: Src/IronPythonTest/Cases/CaseExecuter.cs (hunk 1 above)
Comment (slide) [inline Src/IronPythonTest/Cases/CaseExecuter.cs]: Yes, you are absolutely correct!

File: .gitattributes (hunk 2 above)
Comment (slide) [inline .gitattributes]: Thanks

**Pull request merged**